src/apis/router_console_api.erl

```diff
@@ -157,6 +160,70 @@ get_org(OrgID) ->
         end
     ).
 
+-spec get_orgs() -> {ok, list()} | {error, any()}.
+get_orgs() ->
+    {Endpoint, Token} = token_lookup(),
+    Url = <<Endpoint/binary, "/api/router/organizations">>,
+    lager:debug("get ~p", [Url]),
+    Opts = [
+        with_body,
+        {pool, ?POOL},
+        {connect_timeout, timer:seconds(2)},
+        {recv_timeout, timer:seconds(2)}
+    ],
+    Start = erlang:system_time(millisecond),
+    case hackney:get(Url, [{<<"Authorization">>, <<"Bearer ", Token/binary>>}], <<>>, Opts) of
+        {ok, 200, _Headers, Body} ->
+            End = erlang:system_time(millisecond),
+            ok = router_metrics:console_api_observe(get_orgs, ok, End - Start),
+            lager:debug("Body for ~p ~p", [Url, Body]),
+            {ok, jsx:decode(Body, [return_maps])};
+        _Other ->
+            End = erlang:system_time(millisecond),
+            ok = router_metrics:console_api_observe(get_orgs, error, End - Start),
+            {error, {get_orgs_failed, _Other}}
+    end.
+
+-spec org_manual_update_router_dc(OrgID :: binary(), Balance :: non_neg_integer()) ->
+    ok | {error, any()}.
+org_manual_update_router_dc(OrgID, Balance) ->
+    {Endpoint, Token} = token_lookup(),
+    Url = <<Endpoint/binary, "/api/router/organizations/manual_update_router_dc">>,
+    lager:debug("get ~p", [Url]),
+    Opts = [
+        with_body,
+        {pool, ?POOL},
+        {connect_timeout, timer:seconds(2)},
+        {recv_timeout, timer:seconds(2)}
+    ],
+    Body = #{
+        organization_id => OrgID,
```

Comment:
Do these need to go into our cache?

Reply:
They get picked up on the other side by the websocket. Once the API made the update it sends us a message telling us about the update there